includes/Product/Hooks.php

```diff
@@ -19,6 +19,180 @@ class Hooks {
     public function __construct() {
         add_action( 'template_redirect', [ $this, 'bulk_product_status_change' ] );
         add_action( 'dokan_bulk_product_status_change', [ $this, 'bulk_product_delete' ], 10, 2 );
+        add_action( 'dokan_store_profile_frame_after', [ $this, 'store_products_orderby' ], 10, 2 );
+        add_action( 'wp_ajax_dokan_store_product_search_action', [ $this, 'store_product_search_action' ], 10, 2 );
+    }
+
+    /**
+     * Callback for Ajax Action Initialization
+     *
+     * @since DOKAN_LITE_SINCE
+     *
+     * @return void
+     */
+    public function store_product_search_action() {
+        global $wpdb;
+
+        $return_result              = array();
+        $return_result['type']      = 'error';
+        $return_result['data_list'] = '<li> ' . __( 'Products not found with this search', 'dokan-lite' ) . ' </li>';
+        $output                     = '';
+        $get_postdata               = $_POST;
```

Comment:
kindly don't use $get_postdata, instead use proper nonce verification and then use wp_unslash and proper sanitize function where necessary